.github/workflows/publish-to-pypi-beta.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - staging
+      - conversion-function
```

Comment:
What is this for?

Reply:
Should be removed before PR merges

Reply:
I was trying to publish a beta build to test but it wasn't working due to the yml settings